tools/specification_migration.py

```diff
@@ -0,0 +1,34 @@
+"""
+This script moves the all json specifications from this repository to the database via PUT calls to the FIA_API
+
+It should be run from the root directory of this repository, and expects 1 command line arg the FIA_API_API_KEY
+"""
+
+import json
+import sys
+from pathlib import Path
+
+import requests
+
+FIA_API_API_KEY = sys.argv[1]
+
+FIA_API_PATH: str = "http://localhost:8000"
+successful_update: int = 200
+auth_headers: json = {"Authorization": f"Bearer {FIA_API_API_KEY}", "accept": "application/json"}
+
+# locate specifications
+specifications_path = Path("rundetection/specifications")
+specifications_list = sorted(specifications_path.glob("*.json"))
+
+
+for file_name in specifications_list:
+    instrument_name = file_name.stem.split("_")[0].upper()
+    with file_name.open(mode="r", encoding="utf-8") as specification:
+        json_spec = json.load(specification)
+
+        response = requests.put(
```

Comment:
I actually believe there is a discrepency between the json files and the instruments that exist in the database, can you print the response code for each instrument.

Just so that when we do run it on prod, we can see if something immediately sticks out as odd.

Reply:
I have re-added the print statement (put Ruff) and applied it to all PUT requests.